maint-lib/stagelib/envglobals.py

```diff
@@ -15,6 +15,7 @@
 # Allow lines to extend beyond 99 char limit by 6 chars to support output formatting to 100 cols
 REQUIRED_ENV_VARS = OrderedDict([
     ('CEPH_VERSION',      'Ceph named version being built (e.g., luminous, mimic)'),
+    ('CEPH_POINT_RELEASE', 'Magically extract from CEPH_VERSION and point to a particular release of Ceph'),
```

Comment:
The required vars are for inputs only and not generated vars. `CEPH_POINT_RELEASE` doesn't belong here.

Reply:
Yes & no. As CEPH_POINT_RELEASE is used in the base code, it have to exist. It's kind a way to double-check this variable have been created properly as all important ones.

Reply:
This section is for verifying inputs only. If it needs verified, then it should be verified in the `exportCephVersionSpecifics()`. Alternately, note that the `getEnvVar()` function verifies the existence itself.